src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MultipleVariableDeclarationsCheck.java

```diff
@@ -44,6 +44,22 @@
  * &lt;module name="MultipleVariableDeclarations"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public void myTest() {
+ *     int mid;
+ *     int high;
+ *     // ...
+ *     int lower, higher; // violation
+ *     // ...
+ *     int value,
+ *         index;  //Violation
+ *     // ...
+ *     int place = ar[6], number = ar[place]; // violation, each to be decalared at a new line
```

Comment:
`int place = mid, ....`

Please make it simple and compilable

Reply:
Changes are done, have a look.

Reply:
Please, have a look.